app/api/schema/ticket_fees.py

```diff
@@ -20,7 +20,7 @@ class Meta:
         inflect = dasherize
 
     id = fields.Integer(dump_only=True)
-    currency = fields.Str(validate=validate.OneOf(choices=PAYMENT_CURRENCY_CHOICES), allow_none=True)
-    country = fields.String(allow_none=True)
+    currency = fields.Str(validate=validate.OneOf(choices=PAYMENT_CURRENCY_CHOICES), allow_none=False, default='USD')
```

Comment:
Don't default to USD. We need the user to provide the values. There is no UI showing the user that it will default to USD if no value is provided

Reply:
@iamareebjamal So I should use no default value?

Reply:
Yes